packages/flutter/test/cupertino/action_sheet_test.dart

```diff
@@ -10,8 +10,7 @@ import 'package:flutter_test/flutter_test.dart';
 import '../widgets/semantics_tester.dart';
 
 void main() {
-  testWidgets('Verify that a tap on modal barrier dismisses an action sheet',
-          (WidgetTester tester) async {
+  testWidgets('Verify that a tap on modal barrier dismisses an action sheet', (WidgetTester tester) async {
```

Comment:
Here (and many other places in tests) I put the lambda on the end of line to have:

```dart
testWidgets('xxxx', (WidgetTester tester) async {
  xxxxxx
  xxxxxx
  xxxxxx
});
```

instead of:

```dart
testWidgets('xxxx',
  (WidgetTester tester) async {
    xxxxxx
    xxxxxx
    xxxxxx
  });
```

or:

```dart
testWidgets(
  'xxxx',
  (WidgetTester tester) async {
    xxxxxx
    xxxxxx
    xxxxxx
  },
);
```

This reduces the number of lines changed by this PR.